### DISCUSSION TITLE
Select and insert Clipart

### DISCUSSION THREAD

**Discussion by argostelios** [pubpub-zz/ppInk#109] [Q&A] [answered]
Would be great if  the clipart tool to allow you to select part of your screen and insert that area as a clipart

**Comment by pubpub-zz** [comment, marked as answer]
This is for me working fine : you can take a snapshot through ppInk and use it quickly after pasting from clipboard (it works for me) and if you want to use it more permanently, you can insert through the options dialog and load the image from the snapshots folder (or anywhere else)

**Reply by argostelios** [reply]
Yes it worked for me too thank you very much for the details .
